### DISCUSSION TITLE
Building failed while using mvn clean verify

### DISCUSSION THREAD

**Discussion by AhmadHossain8** [eclipse-platform/eclipse.platform#721] [Q&A]
I am trying to build eclipse from source code but when I write "mvn clean verify" an error occurs. But if I use "mvn clean verify -DskipTests=true" the build is successful. Also I am unsure which jar file is the eclipse IDE because after building it non of the jar files works.

**Comment by laeubi** [comment]
The test require a very specific environment if you really want to execute *all* test, so usually `-DskipTests=true` is a valid option when you only want to build.

Beside that [Eclipse IDE](https://eclipseide.org/) is a product and not a jar file, so don't expect to find a single jar that do anything similar to what you see when starting eclipse.

The best would be to elaborate more what you want to archive, if you really want to build a whole Eclipse you should start with this here: https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/#readme
